pyproject.toml

```diff
@@ -61,3 +81,11 @@ ignore_errors = true
 [[tool.mypy.overrides]]
 module = ["tests._inductor.utils"]
 ignore_errors = true
+
+[tool.uv]
+index-url = "https://download.pytorch.org/whl/cpu"
+extra-index-url = ["https://pypi.org/simple"]
+index-strategy = "unsafe-best-match"
+
+[tool.uv.pip]
+universal = true
```

Comment:
I've been trying to get a set of configurations that renders the same result on my laptop and in CI but it's proving difficult...